ADR/ADRXXX-add-aurora-for-runner.md

```diff
@@ -0,0 +1,21 @@
+# ADRxxx: Add another Aurora cluster to provide database to Forms Runner
```

Comment:
The ADR number in the heading and filename needs to be updated

Reply:
LOL I had made this change locally and forgot to push before marking the PR ready for review... 🫠